docs/sources/enterprise/_index.md

```diff
@@ -50,6 +50,7 @@ With Grafana Enterprise, you get access to new features, including:
 - [Usage insights]({{< relref "usage-insights/_index.md" >}}) to understand how your Grafana instance is used.
 - [Vault integration]({{< relref "vault.md" >}}) to manage your configuration or provisioning secrets with Vault.
 - [Auditing]({{< relref "auditing.md" >}}) tracks important changes to your Grafana instance to help you manage and mitigate suspicious activity and meet compliance requirements.
+- [Request security]({{< relref "request-security.md" >}}) makes it possible to restrict outgoing request from the Grafana server
```

Comment:
```suggestion
- [Request security]({{< relref "request-security.md" >}}) makes it possible to restrict an outgoing request from the Grafana server.
```

Reply:
Good catch, thanks. 

What do you think about making it `requests` instead of adding an "an"?

Reply:
The default in tech writing is to use singular to avoid inconsistency and some complexities. That said, in this context your suggestion makes more sense.

Reply:
I updated my suggestion to plural.

Reply:
Thanks.